### ISSUE TITLE
Any way to make SideBar show Tabs+Windows - like a tab Manager but inside Omni

### ISSUE THREAD

**Issue by xbliss** [Quicksaver/OmniSidebar#105]
Any way to make SideBar show Tabs+Windows - like a tab Manager but inside Omni

It would be the best tab + window manager.

**Comment by Quicksaver** [comment]
I don't think this really fits the scope of the add-on. It's a very complex feature to add, and it'll be very low on my priorities list. So I doubt I will ever get to it. But I'll still keep this here as a suggestion, maybe I'll add it in the future, who knows.

You also might want to try [Tree Style Tab](https://addons.mozilla.org/firefox/addon/tree-style-tab/).
